static/js/reactions.js

```diff
@@ -109,6 +115,51 @@ $(document).on('click', '.reaction-popover-reaction', function () {
 $(document).on('input', '.reaction-popover-filter', filter_emojis);
 $(document).on('keydown', '.reaction-popover-filter', maybe_select_emoji);
 
+exports.reaction_navigate = function (e, event_name) {
+    var first_emoji = get_emoji_from_index(0);
+    var selected_emoji = $(".reaction-popover-reaction").filter(":focus")[0];
+    var reaction_show_list = $(".reaction-popover-reaction").filter(function () {
+        return this.style.display === "block" || this.style.display === "";
+    }).toArray();
+    var selected_index = reaction_show_list.findIndex(function (emoji) {
```

Comment:
See my other comment about possibly building `reaction_show_list` when we do rendering/hiding.  It might make sense to even have a tiny class that also lets you do `find_index_for_emoji` and `get_emoji_at_index`.